### PR TITLE
Seed Database Complete

### PR DESCRIPTION
**Pull request by christina003**
Script to seed database: npm run db:setup

All seed databases files in database

### CODE REVIEW

File: database/Item.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    price: Number,
+    imageUrl: String,
+    description: String,
+    details: [String],
+    availability: Boolean,
+  },
+  seller: String,
+  shippingStatus: String,
+  favorite: Boolean,
+});
```
Comment (kimericbyung) [inline database/Item.js:19]: hey great job here looks neat!

File: database/seed.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        seller: newSeller,
+        shippingStatus: newShippingStatus,
+        favorite: newFavorite,
+      },
+    );
+
+    i += 1;
+  }
+  return items;
+};
```
Comment (kimericbyung) [inline database/seed.js]: The module says that we need 100 primary items.  You only have 5!

**Review by kimericbyung — CHANGES_REQUESTED** [review]
Overall great.  Make sure to add 100 primary keys.

File: database/seed.js (hunk 2 above)
Comment (christina003) [reply database/seed.js]: Changed to 100

**Review by kimericbyung — APPROVED** [review]

**Pull request merged**